agent/app/agent_unspecified.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux,!windows
```

Comment:
Not sure if we still need this file, as currently we only support windows and linux.

Reply:
i develop on my mac. being able to build on all platforms is still a useful thing to adhere to